src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7240/serve.immediately.when.if-none-match.missing.while.polling/accept/client.rpt

```diff
@@ -0,0 +1,127 @@
+property response2Date ${http_cache:datePlus(5)} # sharedProperty
+property etag ${http_cache:strongEtag()} #shared
+
+connect await ROUTED_PROXY
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/data")
+                              .header(":authority", "localhost:8080")
+                              .header("prefer", "wait=5")
+                              .build()}
+connected
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-length", "11")
+                             .header("date", response1Date)
+                             .header("surrogate-control", "max-age=10")
+                             .header("cache-control", "max-age=10")
+                             .header("etag", etag)
+                             .header("preference-applied", "wait=5")
+                             .header("access-control-expose-headers", "preference-applied")
+                             .header("access-control-expose-headers", "etag")
+                             .build()}
+
+read "response #1"
+
+read closed
+
+read notify FIRST_REQUEST_COMPLETED
+
+connect await FIRST_REQUEST_COMPLETED
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/data")
+                              .header(":authority", "localhost:8080")
+                              .header("prefer", "wait=5")
+                              .header("cache-control", "max-age=0")
+                              .header("if-none-match", etag)
+                              .build()}
+connected
+
+write close
+
+write notify SECOND_REQUEST_SENT
+read await CACHED_RESPONSE_EXPIRED
+read await THIRD_REQUEST_SENT
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "304")
+                             .header("etag", etag)
+                             .header("preference-applied", "wait=5")
+                             .header("access-control-expose-headers", "preference-applied")
+                             .header("access-control-expose-headers", "etag")
+                             .build()}
+
+read closed
```

Comment:
Let's simplify to remove response to this `prefer: wait=N` request, and also increase the `wait` to say `86400` so it does not respond during the test scenario.